File: Go/uri_1036.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c, d float64
	fmt.Scan(&a, &b, &c)
	d = math.Sqrt(math.Pow(b, 2) - 4*a*c)
	if d < 0 || a == 0 || math.IsNaN(d) == true {
		fmt.Printf("Impossivel calcular\n")
	} else {
		fmt.Printf("R1 = %.5f\nR2 = %.5f\n", (-b+d)/(2*a), (-b-d)/(2*a))
	}
}

File: Python/uri_1036.py
import math

a, b, c = input().split(' ')
a = float(a)
b = float(b)
c = float(c)
x = (b ** 2 - 4 * a * c)
if x < 0 or a <= 0:
    print('Impossivel calcular')
else:
    d = math.sqrt(x)
    r1 = ((-b + d) / (2 * a))
    r2 = ((-b - d) / (2 * a))
    print('R1 = %0.5f' % r1)
    print('R2 = %0.5f' % r2)
